x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/operator/OperatorPrivilegesSingleNodeTests.java

```diff
@@ -63,20 +67,52 @@ protected Settings nodeSettings() {
         return builder.build();
     }
 
-    public void testOutcomeOfSuperuserPerformingOperatorOnlyActionWillDependOnWhetherFeatureIsEnabled() {
+    public void testOperatorOnlyActionOrSettingWillNotBeActionableByNormalSuperuser() {
```

Comment:
What value do we get from combining these into 1 test case? It seems more natural to have 2 separate cases for action and settings.

Reply:
No real value. I split it and also the corresponding one that test the operator user.